example/index.js

```diff
@@ -28,6 +30,8 @@ var App = React.createClass({
   },
 
   onChange: function(color) {
+    console.log('onChange: ', color);
```

Comment:
It's on the example page and helpful to see what the onChange output looks like when evaluating this component no?